content/administration/maintain/azure_oauth.rst

```diff
@@ -188,24 +188,101 @@ configuration automatically loads the :guilabel:`token` in Odoo, and a tag stati
 Finally, click :guilabel:`Test Connection`. A confirmation message should appear. The Odoo database
 can now send safe, secure emails through Microsoft Outlook using OAuth authentication.
 
-Multiple user configuration
-~~~~~~~~~~~~~~~~~~~~~~~~~~~
+.. _azure_oauth/notifications:
 
-Each user should have a separate server set up. The :guilabel:`from-filter` should be set so that
-only the user's email is sent from that server. In other words, only a user with an email address
-that matches the set :guilabel:`from-filter` is able to use this server.
+Configuration with a single outgoing mail server
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-After setting the :guilabel:`from-filter`, set up a fallback email account to allow for the sending
-of :guilabel:`notifications`. The fallback email must be configured as a :guilabel:`general
-transactional server`. The :guilabel:`mail.default.from` system parameter must be set to the
-:guilabel:`username` of the general transactional server account. For more information see
-:ref:`Use a default email address <email_communication/default>`.
+Configuring a single outgoing server is the simplest configuration available for Microsoft Azure
+and it doesn't require extensive access rights for the users in the database.
+
+A generic email address would be used to send emails for all users within the database. For example
+it could be structured with a `notifications` alias (`notifications@example.com`) or `contact` alias
+(contact@example.com). This address must be set as the :guilabel:`FROM Filtering` on the server.
+This address must also match the `{mail.default.from}@{mail.catchall.domain}` key combination in the
+system parameters.
+
+.. seealso::
+   For more information on the from filter visit :ref:`this documentation
+   <email_communication/default>`.
```

Comment:
I think this may be gray area, but I prefer to stay away from ambiguous link titles (similar to alt text on images). I would rather the link title to be descriptive. By removing the label we can just use the header the anchor is linking to, or provide a more descriptive label than "this documentation". What do you think? I will suggest this for all ambiguous link labels 😉
```suggestion
.. seealso::
   For more information on the from filter visit: :ref:`email_communication/default`.
```

Reply:
I prefer this too.